torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14121,6 +14131,15 @@ def ref_pairwise_distance(input1, input2):
             ),
         ),
     ),
+    OpInfo(
```

Comment:
Ug, this file has gotten too big and it's time for a refactor. But not an issue for this PR, of course

Reply:
IIRC there was some effort from @antocuni to do that. Any progress on that?

Reply:
His work is good it just requires syncing on my end